dev-manual/api/api-intro.rst

```diff
@@ -0,0 +1,118 @@
+.. _api-intro:
+
+============
+API Overview
+============
+
+AtoM includes an API allowing access to part of the AtoM dataset in JSON
+format. The API is implemented in a Symfony plugin that needs to be enabled in
+Admin/Plugins.
+
+.. image:: images/enable-api-plugin.*
+   :align: center
+   :width: 70%
+   :alt: An image of the API plugin in AtoM
+
+.. SEEALSO::
+
+   For more information on available plugins in AtoM, see:
+   :ref:`manage-plugins`. For more information on Symonfy 1.x plugin
+   development, consult the Symfony Project
+   `documentation <http://symfony.com/legacy/doc>`__; in particular, see this
+   section on Plugins:
+
+   * http://symfony.com/legacy/doc/jobeet/1_4/en/20?orm=Propel
+
+There are three endpoints available:
+
+* :ref:`Browse taxonomy terms <api-browse-taxonomies>`
+* :ref:`Browse information objects <api-browse-io>`
+* :ref:`Read information object <api-read-io>`
+
+.. _api-intro-auth:
+
+Authentication
+==============
+
+The AtoM API uses Basic HTTP Auth and Cookies to give access to the data only
```

Comment:
@sevein ok, I've added the following in https://github.com/artefactual/atom-docs/commit/30ef406437988a1177b545f09c72cffd5c4081e2, below the example auth request and response: 

"Cookie-based authentication is optional. It can be used in subsequent requests, or the Authorization header can be re-sent." - does this address it?